### ISSUE TITLE
Ersätta fstil.sty med fstildok.cls

### ISSUE THREAD

**Issue by gyback** [Fysiksektionen/latex-fstildok#1]
Vilka fördelar och nackdelar finns det att slå ihop båda och skapa en guide till den gemensamma?
Just nu är den här utdaterad men det verkar inte vara så svårt att uppdatera den med senaste versionen av fstil och baka in den i en installations program.

**Comment by antolu** [comment]
Tror att

**Comment by antolu** [comment]
Tror att @eringh jobbade på just att baka ihop gamla fstyrdok med fstil. Sen råkade visst Calle ta bort hans access.

**Comment by gyback** [comment]
Jo, Jag ser det påbörjade arbetet men funderar på om vi ska plocka upp det.

**Comment by eringh** [comment]
Min tanke va att det torde vara lämpligt om man hade samma grafiska profil även på styrdokumenten. Då va en merge mellan dessa två klasser ett naturligt val.

Dessutom fanns det en del kod som ligger i båda.

I ytterligare ett steg handlade om att kunna ha en klass som tar hand om även typ motioner etc. Men det kanske inte är någon bra idé.

**Comment by gyback** [comment]
Det låter bra.
Jag plockar upp det och gör en merge på båda.
Funderar på namnet. Ska vi behålla det som fstildok eller döpa det till fstil och avveckla den nuvarande fstil?
